### ISSUE TITLE
bug: YouTube crashes when entering revanced extended settings

### ISSUE THREAD

**Issue by ayushTNM** [inotia00/ReVanced_Extended#364]
### Type

Error at runtime

### Application

Tested on YouTube 18.08.34 and 18.08.36 with patches 2.164.10

### Bug description

Opened revanced extended settings after installing patches using revancify

### Steps to reproduce

Usual patch
Open YouTube
Open revanced extended settings

### Relevant log output

```shell
--------- beginning of crash
03-01 09:47:00.022 E/AndroidRuntime(18332): FATAL EXCEPTION: main
03-01 09:47:00.022 E/AndroidRuntime(18332): Process: app.rvx.android.youtube, PID: 18332
03-01 09:47:00.022 E/AndroidRuntime(18332): java.lang.IncompatibleClassChangeError: Found interface app.revanced.integrations.utils.LogHelper$LogMessage, but class was expected (declaration of 'app.revanced.integrations.utils.LogHelper$LogMessage' appears in /data/app/~~sLZIPfEacOydungWCtUSzg==/app.rvx.android.youtube-W0asbTgNr6x5f9FQcpT1vw==/base.apk!classes7.dex)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at app.revanced.integrations.utils.LogHelper$LogMessage.-$$Nest$mfindOuterClassSimpleName(LogHelper.java:1)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at app.revanced.integrations.utils.LogHelper.printException(LogHelper.java:3)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at app.revanced.integrations.settingsmenu.ReVancedSettingsFragment.onCreate(ReVancedSettingsFragment.java:150)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.Fragment.performCreate(Fragment.java:2486)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1261)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.FragmentManagerImpl.addAddedFragments(FragmentManager.java:2431)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2210)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2166)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2067)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.FragmentManagerImpl.dispatchMoveToState(FragmentManager.java:3057)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:3004)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.FragmentController.dispatchActivityCreated(FragmentController.java:185)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.Activity.performCreate(Activity.java:8313)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.Activity.performCreate(Activity.java:8284)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1417)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3626)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3782)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:101)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2307)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.os.Handler.dispatchMessage(Handler.java:106)
03-01 09:47:00.022 E/AndroidRuntime(18332): 	at android.os.Looper.loopOnce(Looper.java:201)
03-01 09:47:00.022 E/AndroidRuntime(18332):
```

### Screenshots or videos

https://user-images.githubusercontent.com/78740080/222102161-dc49c76d-b724-4d8e-9396-5bac5143a110.mp4

### Additional context

First time logging, so might be wrong

### Device Environment

Android 13 stock

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by inotia00** [comment]
I need a list of patches that you include or exclude

Please attach a screenshot of the patch selection screen in Revancify

Also, it is recommended to include all patches until the issue is resolved.

**Comment by ayushTNM** [comment]
![Screenshot_20230301-114008.png](https://user-images.githubusercontent.com/78740080/222116373-460d1834-d7b8-4f17-8d3e-1f2f186b486f.png)

![Screenshot_20230301-114056.png](https://user-images.githubusercontent.com/78740080/222116400-dda1f340-0c3d-4e23-bb81-ac2945351c95.png)

Should I include all patches or revancify's recommended?

**Comment by ILoveOpenSourceApplications** [comment]
> Tested on YouTube 18.08.34 and 18.08.36 with patches 2.164.10

Could it be because these two versions aren't supported and the latest supported version isn't available in apkmirror yet?

**Comment by DrimTim2020** [comment]
> I need a list of patches that you include or exclude
> 
> Please attach a screenshot of the patch selection screen in Revancify
> 
> Also, it is recommended to include all patches until the issue is resolved.

      "name": "com.google.android.youtube",
      "patches": [
        "default-video-quality",
        "enable-minimized-playback",
        "force-premium-heading",
        "header-switch",
        "hide-button-container",
        "hide-channel-watermark",
        "hide-create-button",
        "hide-email-address",
        "hide-endscreen-cards",
        "hide-flyout-panel",
        "hide-general-ads",
        "hide-info-cards",
        "hide-mix-playlists",
        "hide-pip-notification",
        "hide-shorts-component",
        "hide-snackbar",
        "hide-startup-shorts-player",
        "hide-suggested-actions",
        "hide-tooltip-content",
        "hide-video-ads",
        "microg-support",
        "optimize-resource",
        "overlay-buttons",
        "return-youtube-dislike",
        "settings",
        "sponsorblock",
        "swipe-controls",
        "translations"

**Comment by ayushTNM** [comment]
> 
> > Tested on YouTube 18.08.34 and 18.08.36 with patches 2.164.10
> 
> Could it be because these two versions aren't supported and the latest supported version isn't available in apkmirror yet?

Will test when supported version releases.

**Comment by DrimTim2020** [comment]
> YouTube 18.08.34

I have this YouTube version running on 18.08.34 with previous patches.
And with the new ones (v2.164.10) it already gives an error or throws it on the YouTube start page, or ends with a critical error.

**Comment by DrimTim2020** [comment]
Hmm, I checked version 18.07.34 even lower and now I can say with confidence that three files that were updated today are to blame for this problem:
revanced-integrations-v0.99.10 
revanced-patches-v2.164.10
revanced-patches-v2.164.10.json

**Comment by inotia00** [comment]
fixed in [revanced-patches-v2.164.11](https://github.com/inotia00/revanced-patches/releases/tag/v2.164.11)
